content/version/2/0/code_of_conduct.pt-br.md

```diff
@@ -0,0 +1,90 @@
++++
+version = "2.0"
+aliases = ["/version/2/0"]
++++
+
+# Código de Conduta para Colaborantes
+
+## Nossa promessa
+
+Nós, como participantes, contribuintes e lideranças, nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

Comment:
Sounds great, thank you. I changed the sentence like you suggested. The only thing I kept was the word "lideranças", as this is the word I chose for "leaders", in order to make the writing more inclusive since it's gender neutral.